site/docs/concepts/connectors.md

```diff
@@ -24,18 +24,18 @@ In addition, to benefit from the [control plane security](../tasks/control-plane
 
-The catalog provides metadata about the asset such as security tags. It also provides connection information to describe how to connect to the data source to consume the data. Fybrik uses the metadata provided by the catalog both to enable seamless connectivity to the data and as input to making policy decisions. The data user is not concerned with any of it and just selects the data that it needs regardless of where the data resides.
+The catalog provides metadata about the asset such as security tags. It also provides connection information to describe how to connect to the data source to consume the data. Fybrik uses the metadata provided by the catalog both to enable seamless connectivity to the data and as input to making data governance policy decisions. The data user is not concerned with any of it and just selects the data that it needs regardless of where the data resides.
 
 Fybrik is not a data catalog. Instead, it links to existing data catalogs using connectors.
 The default installation of Fybrik installs [Katalog](../reference/katalog.md), a built-in data catalog using Kubernetes CRDs used for evaluation. A connector to [ODPi Egeria](https://www.odpi.org/projects/egeria) is also available.
 
 ### Policy manager
 
 Enforcing data governance policies requires a Policy Decision Point (PDP) that dictates what enforcement actions need to take place.
-Fybrik supports a wide and extendible set of enforcement actions to perform on data read, write or copy. These include transformation of data, verification of the data, and various restrictions on the external activity of an application that can acceess the data.
+Fybrik supports a wide and extendible set of enforcement actions to perform on data read, copy, (future) write or delete. These include transformation of data, verification of the data, and various restrictions on the external activity of an application that can acceess the data.
 
 A PDP returns a list of enforcement actions given a set of policies and specific context about the application and the data it uses. 
 Fybrik includes a PDP that is powered by [Open Policy Agent](https://www.openpolicyagent.org/) (OPA). However, the PDP can also use external policy managers via connectors, to cover some or even all policy types. 
 
-Policies are therefore defined externally in the policy manager of choice. Fybrik provides a package to help writing data policies in OPA. Otherwise, data stewards are expected to keep using the policy manager that they already use, as long as there is a connector to it.
+Policies are therefore defined externally in the data governance policy manager of choice. Fybrik provides a package to help writing data policies in OPA. Otherwise, data stewards may continue using the data governance policy manager that they already use, assuming there is a Fybrik connector for it.
```

Comment:
This is not clear ('already' is misleading). Sounds as if Fybrik provides a package to write policies in OPA on top of what is used already. I think we no longer support several policy managers.

Reply:
fixed